pkgs/development/python-modules/authheaders/default.nix

```
@@ -11,12 +11,12 @@ buildPythonPackage rec {
     sha256 = "sha256-90rOvu+CbHtammrMDZpPx7rIboIT2X/jL1GtfjpmuOk=";
   };
 
-  propagatedBuildInputs = [ authres dnspython dkimpy publicsuffix2 ]
-                          ++ lib.optional isPy27 ipaddress;
+  propagatedBuildInputs = [ authres dnspython dkimpy publicsuffix2 ];
 
   meta = {
```

```suggestion
  meta = with lib; {
```
otherwise maintainers is undefined